plugins/PagePerformance/Columns/Base.php

```diff
@@ -18,9 +20,30 @@
 abstract class Base extends ActionDimension
 {
     protected $type = self::TYPE_DURATION_MS;
+    protected $columnName;
 
     abstract public function getRequestParam();
 
+    public function getConfigValueCap()
```

Comment:
```suggestion
    private function getConfigValueCap()
```

Reply:
Does private work when the method is called from inherited classes?

Reply:
We can go for `private` here as `Base->getConfigValueCap` gets only called by `Base->getSqlCappedValue`.

If we need to call `Base->getConfigValueCap` from functions _declared in_ an inherited class, we could use `protected` instead of `private`.

Reply:
Yep, `private` here is fine, checked the usages. The other method needs to stay public as it's called from other classes, too.